tools/proto-compiler/src/constants.rs

```diff
@@ -19,7 +19,8 @@ pub const TENDERMINT_VERSIONS: &[TendermintVersion] = &[
     TendermintVersion {
         repo: "https://github.com/celestiaorg/celestia-core",
         ident: "v0_34",
-        commitish: "v0.34.x-celestia",
+        // TODO: update
+        commitish: "a1b07a1e6c77595466da9c61b37c83b4769b47e7", // "v0.34.x-celestia",
```

Comment:
For my curiosity, why was [this particular commit](https://github.com/celestiaorg/celestia-core/commit/a1b07a1e6c77595466da9c61b37c83b4769b47e7) chosen?

Reply:
haha

Reply:
![image](https://github.com/eigerco/celestia-tendermint-rs/assets/34972409/2736aca0-0a72-4173-b9dc-0d9131f77e8c)

Reply:
so @oblique entered a branch name there and this is the commit that happen to be there were the current definitions got generated

Reply:
#1